Sources/Apodini/Semantic Model Builder/gRPC/GRPCService/GRPCServiceUnary.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  GRPCServiceUnary.swift
+//  
+//
+//  Created by Moritz Schüll on 20.12.20.
+//
+
+import Foundation
+@_implementationOnly import Vapor
+
+// MARK: Unary request handler
+extension GRPCService {
+    func createUnaryHandler(requestHandler: EndpointRequestHandler<GRPCInterfaceExporter>)
+    -> (Vapor.Request) -> EventLoopFuture<Vapor.Response> {
+        { (request: Vapor.Request) in
+            let promise = request.eventLoop.makePromise(of: Vapor.Response.self)
+            request.body.collect().whenSuccess { _ in
+                let response: EventLoopFuture<Encodable> = requestHandler.callAsFunction(request: request)
```

Comment:
Small nit, the point of `callAsFunction` is that you can use a RequestHandler instance like the following. This was a follow up change introduced in #85.

For [reference](https://docs.swift.org/swift-book/ReferenceManual/Declarations.html#ID622).

```suggestion
                let response: EventLoopFuture<Encodable> = requestHandler(request: request)
```